packages/core/typings/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@strapi/typings",
+  "version": "4.13.6",
+  "description": "An open source headless CMS solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier. Databases supported: MySQL, MariaDB, PostgreSQL, SQLite",
```

Comment:
This could be better suited to the package.

Reply:
What do you mean ?

Reply:
well the typings package description should be about the package imo e.g. "typings for strapi to be used across internal packages" etc.

Reply:
Ha yes of course !